src/schema/v2/object_identification.ts

```diff
@@ -218,7 +218,7 @@ export const InternalIDFields: GraphQLFieldConfigMap<any, ResolverContext> = {
   internalID: {
     description: "A type-specific ID likely used as a database ID.",
     type: new GraphQLNonNull(GraphQLID),
-    resolve: ({ _id }) => _id,
+    resolve: ({ id }) => id,
```

Comment:
This seems a little dangerous as it would apply to all types that use this, is this correct across the board?

Reply:
Or perhaps, as related to what @ashfurrow mentioned, was it wrong across the board?

Reply:
The transformations were holistically changing `id` to `internalID` for anything that had this resolver. It's always supposed to be `id` if it uses this object id type.

Reply:
Aha, ok, so that was then my bad. Gotcha, thanks! In that case, we should probably revert #1857, right?

Reply:
We could, yeah.

Reply:
Ok, let’s include that here then